ports/ggmsg/portfile.cmake

```
@@ -0,0 +1,99 @@
+    FILENAME "version-20251207.tar.gz"
+    SHA512 bceb8b327656878e4a3b0e2d66cb6e7ae8eb1c7e682bbd03750cc3d0b104f812543cb0ffbcc1fa0575c9d52e2712267dd00e00b59b82f7bca04eca773f82dd41
+)
+
+vcpkg_extract_source_archive_ex(
+    OUT_SOURCE_PATH SOURCE_PATH
+    ARCHIVE "${ARCHIVE}"
+    # (Optional) A friendly name to use instead of the filename of the archive (e.g.: a version number or tag).
+    # REF 1.0.0
+    # (Optional) Read the docs for how to generate patches at:
+    # https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/patching.md
+    # PATCHES
+    #   001_port_fixes.patch
+    #   002_more_port_fixes.patch
+)
+
+# # Check if one or more features are a part of a package installation.
+# # See /docs/maintainers/vcpkg_check_features.md for more details
+# vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+#   FEATURES
+#     tbb   WITH_TBB
+#   INVERTED_FEATURES
+#     tbb   ROCKSDB_IGNORE_PACKAGE_TBB
+# )
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    # OPTIONS -DUSE_THIS_IN_ALL_BUILDS=1 -DUSE_THIS_TOO=2
+    # OPTIONS_RELEASE -DOPTIMIZE=1
+    # OPTIONS_DEBUG -DDEBUGGABLE=1
+)
+
+vcpkg_cmake_build()
+
+# Install header files
+file(COPY "${SOURCE_PATH}/ggmsg/include/" DESTINATION "${CURRENT_PACKAGES_DIR}/include/ggmsg")
+
+# Install release libraries
+if(EXISTS "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/ggmsg.dll")
+    file(COPY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/ggmsg.dll" DESTINATION "${CURRENT_PACKAGES_DIR}/bin/")
+    file(COPY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/ggmsg.lib" DESTINATION "${CURRENT_PACKAGES_DIR}/lib/")
+elseif(EXISTS "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/libggmsg.so")
+    file(COPY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/libggmsg.so" DESTINATION "${CURRENT_PACKAGES_DIR}/lib/")
+elseif(EXISTS "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/libggmsg.dylib")
+    file(COPY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/libggmsg.dylib" DESTINATION "${CURRENT_PACKAGES_DIR}/lib/")
+endif()
+
+# Install debug libraries
+if(EXISTS "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/ggmsg.dll")
+    file(COPY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/ggmsg.dll" DESTINATION "${CURRENT_PACKAGES_DIR}/debug/bin/")
+    file(COPY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/ggmsg.lib" DESTINATION "${CURRENT_PACKAGES_DIR}/debug/lib/")
+elseif(EXISTS "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/libggmsg.so")
+    file(COPY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/libggmsg.so" DESTINATION "${CURRENT_PACKAGES_DIR}/debug/lib/")
+elseif(EXISTS "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/libggmsg.dylib")
+    file(COPY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/libggmsg.dylib" DESTINATION "${CURRENT_PACKAGES_DIR}/debug/lib/")
+endif()
+
+# Install the license file
+if(EXISTS "${SOURCE_PATH}/LICENSE")
+    vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
+endif()

```

```suggestion
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
```

This one seems broken: The source path should always have a license, and if it does not, it needs to fail, not silently try to pass.